templates/vue/src/components/modals/NodeModal/forms/ContentForm/MultipleChoiceForm.vue

```diff
@@ -0,0 +1,274 @@
+<template>
+  <div>
+    <b-form-group
+      :label="
+        multipleAnswerSelected ? 'Multiple Answers Form' : 'Single Answer Form'
+      "
+    >
+      <b-form-group>
+        <b-form-checkbox v-model="useImages" data-qa="multiplechoice-thumbnail">
+          Use Images
+        </b-form-checkbox>
+      </b-form-group>
+      <b-form-group v-if="multipleAnswerSelected">
+        <sortable-list
+          v-model="choiceRows"
+          lockAxis="y"
+          :useDragHandle="true"
+          @input="updateOrderingCheckBoxArray"
+        >
+          <b-form-checkbox-group
+            v-model="preSelectedCheckBoxOptions"
+            class="checkboxGroup"
+          >
+            <choice-row
+              v-for="(choiceRow, index) in choiceRows"
+              :key="choiceRow.id"
+              :data-qa="`choicerow-checkbox-${choiceRow.id}`"
+              :item="choiceRow"
+              placeholder="answer option text"
+              :index="index"
+              :question="question"
+              :multipleAnswerSelected="multipleAnswerSelected"
+              :useImages="useImages"
+              :removeButtonDisabled="isRemoveButtonDisabled"
+              @remove="removeChoiceRowCheckbox(index, choiceRow)"
+            ></choice-row>
+          </b-form-checkbox-group>
+        </sortable-list>
+        <b-button class="addButton" variant="primary" squared @click="addNewChoice">
+          Add a choice
+        </b-button>
+        <p class="message">
+          Tick any options that should be selected by default in the question
+        </p>
+      </b-form-group>
+
+      <b-form-group v-else-if="!multipleAnswerSelected">
```

Comment:
As mentioned in other files as well, we can avoid using if-else here by using `<component` with the `is` prop and computer properties.